tests/dags/test_dag_with_no_tags.py

```diff
@@ -27,5 +27,5 @@
     "start_date": DEFAULT_DATE,
 }
 
-with DAG(dag_id="test_only_dummy_tasks", default_args=default_args, schedule_interval='@once') as dag:
+with DAG(dag_id="test_dag_with_no_tags", default_args=default_args, schedule_interval='@once') as dag:
```

Comment:
There is a `test_only_dummy_tasks` DAG in `test_only_dummy_tasks.py`. Probably a copypasta error. DAG ID is not relevant here; the test this DAG serves never checks it, only the DAG’s content.